custom/ewsghana/reports/specific_reports/reporting_rates.py

```diff
@@ -54,14 +58,16 @@ def charts(self):
             chart_data = [
                 dict(value=reported_percent,
                      label=_('Reported'),
-                     description=_("%.2f%% (%d) Reported (last 7 days)" % (reported_percent, data['total']))),
+                     description=_("%.2f%% (%d) Reported (last 7 days)" % (reported_percent, data['total'])),
+                     color='green'),
```

Comment:
I realize you didn't change this, but translation should be done before formatting.
```python
_("%.2f%% (%d) Reported (last 7 days)") % (reported_percent, data['total'])
```
Same applies below.